src/eve/concepts.py

```diff
@@ -211,12 +224,11 @@ def iter_children(self) -> Generator[Tuple[str, Any], None, None]:
             if not (name.endswith(_EVE_NODE_ATTR_SUFFIX) or name.endswith(_EVE_NODE_IMPL_SUFFIX)):
                 yield name, getattr(self, name)
 
-    # todo(egparedes): disable since unused for now
-    # def select(self, *, kind: Optional[FieldKind] = None) -> Generator[Tuple[str, Any], None, None]:
-    #    for name, _ in self.__fields__.items():
-    #        if not (name.endswith(_EVE_NODE_ATTR_SUFFIX) or name.endswith(_EVE_NODE_IMPL_SUFFIX)):
-    #            if kind and self.__metadata__.get("kind", None) == kind:
-    #                yield name, getattr(self, name)
+    def select(self, *, kind: Optional[FieldKind] = None) -> Generator[Tuple[str, Any], None, None]:
```

Comment:
Fine with me, but am I overlooking something or is this unused? If this stays disabled it would become visible simultanously. Otherwise at least a docstring would be nice.

Reply:
It is unused now. It will be used in the future to select different kinds of fields (input/output). I agree it shouldn't have been added yet, but removing it now could be even more confusing.